traffic_ops/testing/api/v4/tenants_test.go

```diff
@@ -103,6 +108,23 @@ func CreateTestTenants(t *testing.T) {
 	}
 }
 
+func GetTestTenantsByActive(t *testing.T) {
+	opts := client.NewRequestOptions()
+	for _, ten := range testData.Tenants {
+		opts.QueryParameters.Set("active", strconv.FormatBool(ten.Active))
+		resp, reqInf, err := TOSession.GetTenants(opts)
+		if len(resp.Response) < 1 {
+			t.Fatalf("Expected atleast one Tenants response %v", resp)
```

Comment:
This doesn't need to be Fatal; you're never accessing the response again

Reply:
updated to errorf